lib/payload/scanner/header_blind_sqli/engine.py

```diff
@@ -39,5 +39,5 @@ def header_bsqli(host, payloads_bsqli = None, headers_bsqli = None):
         else:
             return False
     except Exception as e:
-        print e
+        print(e)
```

Comment:
I think that this module is completely written in python2.x So in python2.x this is the way we write print statements i.e, without parenthesis. Don't completely depend on automated debugger. They sometimes gives false results. This tool was written in python2.x that's why  "print e" is used instead of print(e).

Reply:
Hey @aman566, I feel the need to correct you here. Python2 has already reached the [end of its life on Jan, 2020](https://www.python.org/doc/sunset-python-2/). So if you still want this framework to be backdated and non-maintained just as Python2, I don't think this suggestion would be really welcome by the mentors. What I did by introducing the function was to add a backport supporting both Python2 and Python3. As indicated by @securestep9, the "original idea behind Nettacker was to **support both versions**", hence this change was necessary. Also If you've gone through the source, all other cases barring this one uses `print()` function instead of only `print`. You yourself have contributed code using `print()`. Right?

Also I don't think all of this code was contributed by some _automated debugger_, or is it? ;)